bindings/cs/rl.net.native/rl.net.live_model.h

```diff
@@ -25,34 +25,36 @@ typedef struct livemodel_context {
 
 // Global exports
 extern "C" {
-    // NOTE: THIS IS NOT POLYMORPHISM SAFE!
-    API livemodel_context_t* CreateLiveModel(reinforcement_learning::utility::configuration* config, factory_context_t* factory_context);
-    API void DeleteLiveModel(livemodel_context_t* context);
+  // NOTE: THIS IS NOT POLYMORPHISM SAFE!
```

Comment:
are these all just whitespace changes?

Reply:
I added 2 new methods here (lines 52,53) 
1. LiveModelReportOutcomeSlotStringIdF
2. LiveModelReportOutcomeSlotStringIdJson

Other than that it is all whitespace changes